src/catalogue/items/obsoleteCatalogueItemDialog.component.tsx

```diff
@@ -325,6 +324,19 @@ const ObsoleteCatalogueItemDialog = (
         <Box sx={{ textAlign: 'center', marginTop: 2 }}>
           {renderStepContent(activeStep)}
         </Box>
+        <Box
```

Comment:
```suggestion
        {formError && (<Box
```
Isn't this still needed to not display it when we don't have it?